lib/clevertap_plugin_web.dart

```diff
@@ -4,18 +4,34 @@ import 'package:clevertap_plugin/src/clevertap_plugin_web_binding.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_web_plugins/flutter_web_plugins.dart';
 import 'package:js/js_util.dart' as js;
+import 'dart:html' as html;
```

Comment:
@KambleSonam Please verify whether the sample app is working fine on both Android and iOS platforms since the last time when `dart:html ` was used, I believe it raised a runtime error on Android.